### PR TITLE
Fix: Issue #2: Removed some syntax errors

### PR DESCRIPTION
**Pull request by avighnac**
Fixes issue https://github.com/AaravMalani/ConstantPool-Editor/issues/2

Or is it  'compilation' errors? I don't know anymore, I don't use Python, but these were pretty much equivalent, since you couldn't really use the application before.

(it's not like it works now but yeah)

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -70,15 +69,13 @@ def nextBytes(no_of_bytes: int, data: bytes, func: function = int.from_bytes) ->
...
     sys.exit(1)
 
-minor, classData = nextBytes(2, classData)
-major, classData = nextBytes(2, classData)
-major_to_string = {k: '1.'+str(i) for k, i in zip([45]+list(range(45, 64)), [*range(0, 20)])} # This is very unintuitive code to generate the mappings for int to string major versions
-if not major_to_string.get(major):
+minor, classData = nextBytes(2, classData, bytes)
+major, classData = nextBytes(2, classData, bytes)
+major_int = int.from_bytes(major, byteorder='big')
+if major_int not in range(45, 64):
```
Comment (AaravMalani) [inline main.py:72]: Why remove the mappings?

File: main.py [hunk 2]
```diff
@@ -70,15 +69,13 @@ def nextBytes(no_of_bytes: int, data: bytes, func: function = int.from_bytes) ->
...
     print("ERROR: Invalid major version! Perhaps try updating this software (Version "+__version__+")", file=sys.stderr)
     sys.exit(1)
-constant_pool_count, classData = nextBytes(2, classData)
-
-
+constant_pool_count, classData = nextBytes(2, classData, bytes)
```
Comment (AaravMalani) [inline main.py:76]: This isn't how the `nextBytes` works and will always return 4 afaik

File: main.py [hunk 3]
```diff
@@ -246,8 +243,8 @@ def pack(self):
         return pack([{self.tag: 1}, {self.bootstrap_method_attr_index: 2}, {self.name_and_type_index: 2}])
 
 
-for i in range(constant_pool_count - 1):
-    tag, classData = nextBytes(1, classData)
+for i in range(len(constant_pool_count) - 1):
+    tag, classData = nextBytes(1, classData, bytes)
```
Comment (AaravMalani) [inline main.py:244]: This causes the value not to be present in the `lengthTable` leading to an error

File: main.py (hunk 1 above)
Comment (avighnac) [reply main.py]: You weren't using them!

**Pull request merged**